oop/src/views/MoveInfoPage.vue

```diff
@@ -29,18 +29,36 @@
       <h2 class='mt-4'>What is the new address information?</h2>
       <div class="row">
         <div class="col-md-6">
-          <Input label='Address line 1'
-                 className='mt-3' />
-          <Input label='Address line 2'
-                 className='mt-3' />
-          <Input label='Country'
-                 className='mt-3' />
-          <Input label='Province/Region/State'
-                 className='mt-3' />
+          <CountryInput label='Country'
+                 className='mt-3'
+                 v-model="country" />
+          <div class="text-danger" v-if="$v.country.$dirty && !$v.country.required" aria-live="assertive">Field is required.</div>
+          <div v-if='country === "CA"'>
+            <Input label='Address line'
+                  className='mt-3'
+                  v-model="addressLine" />
+            <div class="text-danger" v-if="$v.addressLine.$dirty && !$v.addressLine.required" aria-live="assertive">Field is required.</div>
+          </div>
+          <div v-if='country !== "CA"'>
+            <TextArea label='Address line text area'
+                  className='mt-3'
+                  v-model="addressLine"/>
+            <div class="text-danger" v-if="$v.addressLine.$dirty && !$v.addressLine.required" aria-live="assertive">Field is required.</div>
+          </div>
+          <Input label='Province/State/Region'
+                 className='mt-3'
+                 v-model="province" />
+          <div class="text-danger" v-if="$v.province.$dirty && !$v.province.required" aria-live="assertive">Field is required.</div>
           <Input label='City/Town'
-                 className='mt-3' />
-          <Input label='Postal Code/Zip'
-                 className='my-3' />
+                 className='mt-3'
+                 v-model="city" />
+          <div class="text-danger" v-if="$v.city.$dirty && !$v.city.required" aria-live="assertive">Field is required.</div>
+          <PostalCodeInput id="postalCode"
+            label="Postal Code/Zip Code"
```

Comment:
Please update to use `PostalCodeInput` from "common-lib-vue" package. Also, since the `PostalCodeInput` component only accepts postal codes, you can probably remove "Zip Code" from the label.